src/ReportCommand.php

```diff
@@ -141,6 +177,38 @@ protected function outputFormattedLicenses(OutputInterface $output, array $licen
                 ]);
             }
             $licenseTable->render();
+
+            if ($input->getOption('show-packages') || $output->isVerbose()) {
+                if ($out = $this->outputFormatPackages($input, $dependencies)) {
```

Comment:
What's the purpose of this condition? Under which circumstances do we not want to print here?

If it is because we have no dependencies for a given license a more readable version would be:
```php
if (! empty($dependencies)) {
    $output->writeln($this->formatPackageOutput($dependencies, $input->getOption('grouped')));
}
```

Reply:
I think the condition on line 182 is useless. 

But the condition on line 181 is the only check for the option.